### ISSUE TITLE
consistent function names

### ISSUE THREAD

**Issue by mbannert** [mbannert/timeseriesdb#178]
While I agree that `store_time_series` and `read_time_series` as the package most common functions should be named after the accessibility first principle, I think the rest of the db interaction functions should follow the same patterns. This would imply changing some of this: 

- db_update_release
- db_cancel_release

vs. 

- db_collection_add
- db_collection_delete

etc. I am all for going for the second pattern, because it is helpful in auto complete settings, even though it does not read like a sentence anymore. What do you think @HomoCodens @angelicambg
